server/asset/btc/btc.go

```diff
@@ -197,6 +200,9 @@ type BackendCloneConfig struct {
 	// FeeEstimator provides a way to get fees given an RawRequest-enabled
 	// client and a confirmation target.
 	FeeEstimator func(*RPCClient) (uint64, error)
+	// BooleanGetBlockRPC causes the RPC client to use a boolean second argument
+	// for the getblock endpoint, instead of Bitcoin's numeric.
+	BooleanGetBlockRPC bool
```

Comment:
I don't see this being used anywhere.

Reply:
whoops, half checked in changes.

This seemingly pointless field in `Backend` is because `RPCClient` isn't constructed until connect, and the `dexbtc.RPCConfig` is pretty much intended just to be for ini parsing of the .conf RPC settings.

Reply:
https://github.com/decred/dcrdex/compare/e1902e7bd60af9021a622a2e80575d2d94b4bf2f..e859d9fe6307a035f74ac11be1c88428d7991483